app/assets/javascripts/application.js

```diff
@@ -4,3 +4,8 @@
 //= require bootstrap
 //= require bootstrap-notify
 //= require_tree .
+//= require clipboard
+
+$(document).ready(function() {
+  new Clipboard('#clip');
```

Comment:
so there can be only one element per whole page for this. Is it by design?

Imho you should avoid identifiers (#) in css when possible. Some people use `js-` as a prefix for classes to point out that this will be used for javascript handlers.